wordpress-mobile/WordPress-Android
# WP Stories: use wp:jetpack/story block structure

This PR produces a post that contains the agreed GB block structure for alpha1.

Note: in the long run (when the `wp:jetpack/story` block is registered and Stories functionality is integrated with Gutenberg), we should be extending the already existing `MediaUploadCompletionProcessor` class and make it aware of the `wp:jetpack/story` block so things can be processed there accordingly. For alpha 1/2, we're producing the block structure from scratch, and this should suffice.

To test:
1. run the app, and add the test site to your sites
2. create a story
3. publish
4. observe it gets published and the alpha web viewer can display the content properly

PR submission checklist:
